README.md

```diff
@@ -25,13 +24,34 @@ The Biconomy SDK is your all-in-one toolkit for building decentralized applicati
 
 Unlock the full potential of **ERC4337 Account Abstraction** with methods that simplify the creation and dispatch of UserOperations, streamlining dApp development and management.
 
+```javascript
+const biconomyAccount = new BiconomySmartAccount(biconomySmartAccountConfig);
+const biconomySmartAccount =  await biconomyAccount.init();
+console.log("owner: ", biconomySmartAccount.owner);
+console.log("address: ", biconomySmartAccount.address);
+```
+
 ### Bundler
 
 Leverage standardized bundler infrastructure for efficient operation of account abstraction across EVM networks.
 
+```javascript
+const bundler: IBundler = new Bundler({
+    bundlerUrl: '', // From Biconomy Dashboard
+    chainId: ChainId.POLYGON_MUMBAI,
+    entryPointAddress: DEFAULT_ENTRYPOINT_ADDRESS,
+});
+```
+
 ### Paymaster
 
-Acting as third-party intermediaries, Paymasters have the capability to sponsor gas fees for an account, provided specific predefined conditions are met. Additionally, they can accept gas payments in ERC20 tokens from users' smart accounts, with the Paymaster managing the conversion to native tokens for gas payment.
```

Comment:
Why this line is removed?

Additionally, they can accept gas payments in ERC20 tokens from users' smart accounts, with the Paymaster managing the conversion to native tokens for gas payment.